test/views/declaration/additionaldeclarationtype/DeclarationTypeViewSpec.scala

```diff
@@ -115,69 +115,64 @@ class DeclarationTypeViewSpec extends UnitViewSpec with ExportsTestData with Com
       }
     }
 
-    "display 'Save and continue' button" when {
+    "display 'Continue' button" when {
 
       "used for Standard Declaration journey" in {
         val view: Document = createView(formStandard, DeclarationType.STANDARD)
-        view.getElementById("submit").text() mustBe messages(saveAndContinueCaption)
+        view.getElementById("submit").text() mustBe messages(continueCaption)
       }
 
       "used for Supplementary Declaration journey" in {
         val view: Document = createView(formSupplementary, DeclarationType.SUPPLEMENTARY)
-        view.getElementById("submit").text() mustBe messages(saveAndContinueCaption)
+        view.getElementById("submit").text() mustBe messages(continueCaption)
       }
 
       "used for Simplified Declaration journey" in {
         val view: Document = createView(formSimplified, DeclarationType.SIMPLIFIED)
-        view.getElementById("submit").text() mustBe messages(saveAndContinueCaption)
+        view.getElementById("submit").text() mustBe messages(continueCaption)
       }
 
       "used for Occasional Declaration journey" in {
         val view: Document = createView(formOccasional, DeclarationType.OCCASIONAL)
-        view.getElementById("submit").text() mustBe messages(saveAndContinueCaption)
+        view.getElementById("submit").text() mustBe messages(continueCaption)
       }
 
       "used for Clearance Request journey" in {
         val view: Document = createView(formClearance, DeclarationType.CLEARANCE)
-        view.getElementById("submit").text() mustBe messages(saveAndContinueCaption)
+        view.getElementById("submit").text() mustBe messages(continueCaption)
       }
     }
```

Comment:
This bunch of test cases seems like a good place to use `onEveryDeclarationJourney` method

Reply:
I thought about that.  For about 200mS until I realised that the views are created from journey-specific components.   So I just fix what I broke.